File: Pre-Ileya/Day 4/JavaScript/Task2.js
function unique(array){
    let count = 0;

    for (i = 0; i < array.length; i++){
        boolean seenBefore = false;            
        for (j = 0; j < i; j++){
            if (array[i] == array[j]){
                seenBefore = true;
                break;
            }           
        }
        if (!seenBefore) count++; 
    }

    let uniqueSubset = [];

    for (i = 0; i < array.length; i++){
        boolean seenBefore = false;
        for (j = 0; j < i; j++){
            if (array[i] == array[j]){
                seenBefore = true;
                break;
            }
        }

        if (!seenBefore){
            uniqueSubset.append(array[i]) = array[i];
        }
    }  
        
    return uniqueSubset;    
}
}

File: Pre-Ileya/Day 4/Python/task_2.py
def unique(array):
    count = 0;

    for i in range(len(array)):
        seenBefore = false   
        for j in range(i):
            if (array[i] == array[j]):
                seenBefore = true
                break
         
        if !seenBefore:
            count += 1

    uniqueSubset = []

    for i in range(len(array)):
        seenBefore = false   
        for j in range(i):
            if (array[i] == array[j]):
                seenBefore = true
                break

        if !seenBefore:
            uniqueSubset.append(array[i]) = array[i]
 
    return uniqueSubset
